### ISSUE TITLE
Include number of replacements in Report

### ISSUE THREAD

**Issue by josefkarasek** [openshift/must-gather-clean#53]
Current Report:
```
replacements:
  - 10-0-22-225: x-ipv4-000014-x
    10-0-129-220: x-ipv4-000008-x
    10-0-151-183: x-ipv4-000010-x
```
Proposed Report:
```
replacements:
  - 10-0-22-225:
    replacedWith: x-ipv4-000014-x
    total: 12
  - 10-0-129-220: 
    replacedWith: x-ipv4-000008-x
    total: 1
  - 10-0-151-183: 
    replacedWith: x-ipv4-000010-x
    total: 350
```

**Comment by tjungblu** [comment]
This is tracked in our backlog in CFE-131.

**Comment by tjungblu** [comment]
implemented with #65
